picard/formats/apev2.py

```diff
@@ -135,6 +135,8 @@ def _load(self, filename):
         self.__casemap = {}
         file = self._File(encode_filename(filename))
         metadata = Metadata()
+        config = get_config()
+        date_sanitize = not config.setting['disable_date_sanitize']
```

Comment:
Since those variables are only used in `if file.tags` they should be moved inside the branch, just before the `for` loop.